### DISCUSSION TITLE
求个真人720p老片着色器组合

### DISCUSSION THREAD

**Discussion by jiang-yuan** [hooke007/mpv_PlayKit#128] [Q&A 问答]
我试了ctrl+2有一点点感觉但是非常不明显
ctrl+9的组合是针对动画的,看起来很怪

就是有些几年前的电影嘛,补完帧流畅多了但是清晰度太低了,你懂得

**Comment by hooke007** [comment]
真人这种东西。。。暂时没有能质变的，各种锐化都试试，凑活着看吧

**Reply by hooke007** [reply]
社区看上去对此并不感冒，着色器这几年的更新也依然都是anime取向。即使驱动不支持，滤镜的选择也足够丰富。

关闭

**Comment by gejianya** [comment]
补帧我觉着用处不太大，30补到60我没看出来区别。然后就是着色器以前用的是ani4K fast A+A，但是画面动漫化严重，没多大意思，可能是真人超分比较难，不过现在N卡有自带超分辨率技术啊，比播放器里着色器那是好用太多了。
